internal/summary/summary.go

```diff
@@ -70,47 +69,38 @@ func (s *Summary) Gather(services []model.Manifest) error {
 }
 
 func (s *Summary) Output() string {
-	ret := "\n──┤ Services Built … ├────────────────────────────────────────\n"
-
+	ret := ""
 	for _, svc := range s.Services {
-		ret += fmt.Sprintf("    SERVICE NAME: %s\n", svc.Name)
-		ret += fmt.Sprintf("    WATCHING: %s\n", svc.Path)
-
-		k := svc.k8sData
+		indent := " "
+		ret += fmt.Sprintf("%s%s — ", indent, svc.Name)
 
-		ret += fmt.Sprintln("    KUBERNETES INFO")
-		if len(k.Version) > 0 {
-			ret += fmt.Sprintf("      • Version: %s\n", k.Version)
+		// Relative Path
+		wd, err := os.Getwd()
+		if err != nil {
+			log.Fatalf("Failed to get working directory: %s", err)
 		}
-
-		if len(k.Group) > 0 {
-			ret += fmt.Sprintf("      • Group: %s\n", k.Group)
+		rel, err := filepath.Rel(wd, svc.Path)
+		if err != nil {
+			log.Fatalf("Failed to get relative path: %s", err)
 		}
+		ret += fmt.Sprintf("./%s ", rel)
 
+		// K8s info
+		k := svc.k8sData
+		if len(k.Name) > 0 {
+			ret += fmt.Sprintf("→ `kubectl get deploy %s` ", k.Name)
```

Comment:
...i don't understand what's helpful about this string?

Reply:
You mean, why do we say the whole "kubectl get deploy..."? Dan's idea is that this is a hint to our users that they can find out more about the deployed K8s service using that one-liner.